docs/source/release_notes.rst

```diff
@@ -3,11 +3,16 @@ Release Notes
 **Future Releases**
     * Enhancements
         * Updated `pipeline.get_prediction_intervals()` to add trend prediction interval information from STL decomposer :pr:`4093`
+        * Add component-specific nullable type handling :pr:`4043`
```

Comment:
An alternative (and I'm not sure what's better here so please feel free to ignore if you disagree) would be to simply list this PR as a second PR for all the changes - i.e. `* Handled nullable type incompatibility in ``Decomposer`` :pr:`4105`, :pr:`4043` - there is some precedent for this method earlier in these notes!

I will also continue to be the release notes sticker, can all these comments be past tense? 😁

Reply:
I like that! Would be better at helping people find the PRs where the discussions happened if need be! 

(also sorry about making you be a broken record on the release notes thing. To be fair, I apparently cannot remember, but I also think I chose the wrong note in a couple of rebases, undoing your hard work from the past)